packages/report-server/src/reportBuilder/output/functions/matrix/matrix.ts

```diff
@@ -51,24 +51,26 @@ const paramsValidator = yup.object().shape(
             )
           : schema,
       ),
+    dataGroups: yup.array().of(yup.string()).notRequired(),
   },
   [['rowField', 'categoryField']],
 );
 
-const matrix = (rows: Row[], params: MatrixParams): Matrix => {
+export const matrix = (rows: Row[], params: MatrixParams): Matrix => {
```

Comment:
These exports are no longer needed yeah? Lets try and keep things encapsulated 👍